### PR TITLE
add support for shipping dimensions

### PR DESCRIPTION
**Pull request by jakeday**
Google Merchant Center recently started requesting Shipping Length, Width, and Height. This PR adds the ability to add these attributes.

### CODE REVIEW

File: src/Product.php [hunk 1]
```diff
@@ -367,6 +367,45 @@ public function setShippingWeight($str)
         return $this;
     }
 
+    /**
+     * Add shipping length
+     * 
+     * @param string $str
+     *
```
Comment (vitalybaev) [inline src/Product.php:374]: I think it's nice to have a link to a documentation about this field. Something like this:

`@see https://support.google.com/merchants/answer/6324498?hl=en`

And in other places too.

**Review by vitalybaev — COMMENTED** [review]
Nice addition! I just only ask you about one improvement before it'll be merged

**Comment by jakeday** [comment]
Done!

**Review by vitalybaev — APPROVED** [review]

**Comment by vitalybaev** [comment]
Thanks, @jakeday!
Released as `v2.6.0`

**Pull request merged**